dbusmock/mockobject.py

```diff
@@ -232,6 +234,7 @@ def AddObject(self, path, interface, properties, methods):
                              properties)
         # make sure created objects inherit the log file stream
         obj.logfile = self.logfile
+        obj.object_manager = self.object_manager
```

Comment:
I don't understand that bit -- `AddObject()` is not about inheritance or similar, it's a "meta-method" on the mock to create new mocked objects. These should not automatically be assumed to be object managers?